src/rust/grapl-web-ui/tests/api/ingress.rs

```diff
@@ -0,0 +1,41 @@
+#![cfg(feature = "integration_tests")]
+
+use eyre::ContextCompat;
+
+use crate::test_app::TestApp;
+
+#[actix_web::test]
+async fn publish_log() -> eyre::Result<()> {
+    let app = TestApp::init().await?;
+
+    app.login_with_test_user().await?;
+
+    let plugin_name = "Test Plugin";
+
+    let create_response = crate::plugin::create_plugin(&app, plugin_name).await?;
+
+    //TODO: this shouldn't be necessary, but we're seeing 50 errors without it.
+    // I'll file a task to look into it for now so we can unblock frontend work.
+    std::thread::sleep(std::time::Duration::from_secs(5));
```

Comment:
like fifty separate errors or 50x errors?

Reply:
dont let this comment prevent you from shipping though its an easy fix later if it's 50x

Reply:
It bothers me, I can wait for another CI test run. 👍